index.html

```diff
@@ -5,8 +5,33 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/e2597287a1.js" crossorigin="anonymous"></script>
   <title>Portfolio</title>
 </head>
 <body>
+  <div class="app-cont">
+    <div class="header-cont">
+      <div class="nav-cont">
+        <div class="header-img"><img src="images/image_geometry_1.png" alt="">
+        </div>
+        <nav class="nav-bar">
+          <i class="fa-solid fa-bars fa-xl"></i>
+      </div>
+      <header>
+        <h1 class="header-h1">
+          Fuad <br> Nabiyev
+        </h1>
+        <p class="header-p">
+          Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+          If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
+      </header>
+      <div class="start-btn">
+        <h2 class="btn-h2">
+          Start collaboration
+        </h2>
```

Comment:
- [ ] you should change this `<h2>` to `<button>` that make sense than div cause it's a button 
- [ ] Also please try to add font-family and font-size as in the Figma design in order to match it